Mathlib/NumberTheory/Dioph.lean

```diff
@@ -603,11 +617,14 @@ theorem mod_dioph : DiophFn fun v => f v % g v :=
                 exact ⟨or_iff_not_imp_left.2 fun h => mod_lt _ (Nat.pos_of_ne_zero h), x / y,
                   mod_add_div _ _⟩⟩
 
+/-- Diophantine functions are closed under modulo. -/
```

Comment:
I think for many notations it would be even better if you put the docstring on the identifier the notation refers to (like `Dioph.mod_dioph` and then annotated the notation with `@[inherit_doc]`. What do you think?

Reply:
Done.